src/wallet/wallet.h

```diff
@@ -294,6 +294,11 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     // Local time that the tip block was received. Used to schedule wallet rebroadcasts.
     std::atomic<int64_t> m_best_block_time {0};
 
+    static const int64_t BLANK_BIRTH_TIME = -1;
+    // First created key time. Used to skip blocks prior to this time.
+    // 'BLANK_BIRTH_TIME' if wallet is blank.
+    std::atomic<int64_t> m_birth_time{BLANK_BIRTH_TIME};
```

Comment:
In 1a9a349a026af2865e5446091f1e072b7a845819 "wallet: skip block scan if block was created before wallet birthday"

`m_birth_time` could be initialized with `std::numeric_limits<in64_t>::max()` which would remove the need to compare against `BLANK_BIRTH_TIME`.

Reply:
pushed thanks